documentation/source/PoseEstimation.md

```diff
@@ -0,0 +1,187 @@
+
+In order to use `PoseEstimationMetrics` you have to pass a so-called `post_prediction_callback` to the metric, which is responsible for postprocessing of the model's raw output into final predictions. 
+
+### Postprocessing
+
+Postprocessing refers to a process of transforming the model's raw output into final predictions. Postprocessing is also model-specific and depends on the model's output format. 
+For `DEKR` model, the postprocessing step is implemented in [DEKRPoseEstimationDecodeCallback]((https://docs.deci.ai/super-gradients/docstring/training/utils/#training.utils.pose_estimation.dekr_decode_callbacks.DEKRPoseEstimationDecodeCallback)) class. 
+When instantiating the metric, one has to pass a postprocessing callback as an argument:
+
+```yaml
+training_hyperparams:
+    valid_metrics_list:
+      - PoseEstimationMetrics:
+          num_joints: ${dataset_params.num_joints}
+          oks_sigmas: ${dataset_params.oks_sigmas}
+          max_objects_per_image: 20
+          post_prediction_callback:
+            _target_: super_gradients.training.utils.pose_estimation.DEKRPoseEstimationDecodeCallback
+            max_num_people: 20
+            keypoint_threshold: 0.05
+            nms_threshold: 0.05
+            nms_num_threshold: 8
+            output_stride: 4
+            apply_sigmoid: False
+```
+
+### Visualization
+
+Visualization of the model predictions is a very important part of the training process for pose estimation models. 
+By visualizing the predicted poses, developers and researchers can identify errors or inaccuracies in the model's output and adjust the model's architecture or training data accordingly.
+
+Overall, visualization is an important tool for improving the accuracy and usability of pose estimation models, both during development and in real-world applications.
+
+SuperGradients provide an implementation of `DEKRVisualizationCallback` to visualize predictions for `DEKR` model. 
+You can use this callback in your training pipeline to visualize predictions during training. To enable this callback, add the following lines to your training YAML recipe:
+
+
+```yaml
+training_hyperparams:
+  resume: ${resume}
+  phase_callbacks:
+    - DEKRVisualizationCallback:
+        phase:
+          _target_: super_gradients.training.utils.callbacks.callbacks.Phase
+          value: TRAIN_BATCH_END
+        prefix: "train_"
+        mean: [ 0.485, 0.456, 0.406 ]
+        std: [ 0.229, 0.224, 0.225 ]
+        apply_sigmoid: False
+
+    - DEKRVisualizationCallback:
+        phase:
+          _target_: super_gradients.training.utils.callbacks.callbacks.Phase
+          value: VALIDATION_BATCH_END
+        prefix: "val_"
+        mean: [ 0.485, 0.456, 0.406 ]
+        std: [ 0.229, 0.224, 0.225 ]
+        apply_sigmoid: False
+```
+
+## Implementing your own model
```

Comment:
I personally think users would find it more helpful if we would change this section to "how to connect your own dataset".
I believe the vast majority of SG users do not come with their own model, but I might be biased.

Reply:
Added both sections on adding dataset and model